### PR TITLE
Model Downloader: add a machine-readable output format

### PR DESCRIPTION
**Pull request by IRDonch**

### CODE REVIEW

**Comment by IRDonch** [comment]
Don't merge yet; I just noticed some things that need to be fixed.

**Comment by IRDonch** [comment]
False alarm, but I used it to change the name of the option. Ready for review now.

File: tools/downloader/downloader.py [hunk 1]
```diff
@@ -43,53 +43,46 @@ def process_download(chunk_iterable, size, file):
...
             return True
-        except requests.exceptions.ConnectionError as e:
-            print("Error Connecting:", e)
-        except requests.exceptions.Timeout as e:
-            print("Timeout Error:", e)
-        except requests.exceptions.TooManyRedirects as e:
-            print("Redirects Error: requests exceeds maximum number of redirects", e)
-        except (requests.exceptions.RequestException, ssl.SSLError) as e:
-            print(e)
+        except (requests.exceptions.RequestException, ssl.SSLError):
```
Comment (zmaslova) [inline tools/downloader/downloader.py]: why do you remove all other exceptions?

Comment (IRDonch) [reply tools/downloader/downloader.py]: The other exceptions are subclasses of `RequestException`, and their exception messages contain enough detail to make special handling for them unnecessary.

**Pull request merged**